organization/_posts/2018/2018-10-18-coordination.md

```diff
@@ -0,0 +1,156 @@
+        registration soon.
+-   Topics also announced at the GDB yesterday.
+
+New Working Groups
+==================
+-   Mandate was put on the [HSF
+    Website](https://hepsoftwarefoundation.org/organization/working-group-mandates.html).
+-   Call for convenors was renewed, until 29 October.
+    -   Several new nominations received; this is the [up-to-date
+        list](https://docs.google.com/document/d/19-Etynm2gO07PGVGSDETmGNStWX3oq6TIpXftsbMoAc/edit?usp=sharing).
+    -   If there is a need to choose between the nominations, HSF
+        Coordination will be asked to do so.
+-   Stubs for new working groups added to the website.
+
+Generators Software Computing Workshop
+======================================
+-   [Indico agenda](https://indico.cern.ch/event/751693/).
+    -   Timetable has been filled in with a lot more detail on the
+        presentations and confirmed speakers.
+-   33 people registered.
+-   Planning a hackathon event on Wednesday 28th, exact topics to be
+    decided with input from a short survey.
+-   Comments about licensing?
+    -   It’s important, but not the “critical” item.
+    -   TBD, so a topic for the next organisers meeting.
+
+Activity updates
+================
+
+Licensing
+---------
+-   Still hoping to meet with FastJet authors in the week of the
+    Generators Workshop.
+    -   ATLAS, CMS input.
+    -   Input from OpenLab about companies like IBM not agreeing to run
+        at their facilities a code which is GPL.
+
+Training
+--------
+-   Graeme, Davide and James met to discuss ATLAS follow-up after last
+    week’s training focused discussions.
+    -   Definite interest in holding a ‘StarterKit’ like event in
+        advance of the next ATLAS induction week/software tutorial at
+        CERN.
+        -   Dates being followed up.
+    -   Core skills of interest: Git, Shell, Python, C++.
+    -   Following up with Sudhir and Pete for FIRST-HEP collaboration.
+-   Please comment on [David’s training
+    survey](https://docs.google.com/forms/d/1DEb4h49UiJkUZscIemQaIWub_3jVC5Nn49T1dw6s-ws/edit).
+
+Packaging
+---------
+-   Due to a sparse agenda and various clashes, [next
+    meeting](https://indico.cern.ch/event/766022/) will be 7
+    November.
+
+Software Development
+--------------------
+-   Would like to organise a meeting in November on monitoring tools.
+
+Software Forum
+---------------
+-   Next meeting is [HEP analysis in the Numpy
+    ecosystem](https://indico.cern.ch/event/745288/), 7 November.
+-   Topics for next year - please suggest!
+
+PyHEP
+-----
+-   Continuing to make contact with neutrino community.
+-   See Software Forum topic for next meeting!
+-   Website update in the pipeline (Eduardo) to give a better overview
+    of activities.
+
+Technical Notes
+---------------
+-   Eduardo and Andy McNab following up on DOIs for some of the
+    technical notes.
+
+CWP
+===
+-   ### General Matters and Roadmap
```

Comment:
Remove the bullet marker on this line. And see my general remark about mixing the heading markup styles.

Reply:
Sure, no problem, but AFAICS this is the markdown we have used for ages in this section, e.g., from

https://hepsoftwarefoundation.org/organization/2018/10/04/coordination.html

```
CWP
===
-   ### General Matters and Roadmap
    -   [*CSBS review
        comments*](https://docs.google.com/document/d/16T2RRu1LmAyXTgtKjyWgOwZR8zsVOw2Y1SCntot3_NU/edit?usp=sharing)
    -   Michel was too occupied to make progress on these yet (seems
        everyone else was too).
-   ### Publication status for Individual WG Papers
    -   **Software Trigger and Event Reconstruction**
        -   PR to the documents repo was
```

Changing from `===` to `#` will change nothing in the CWP title (I did try it). The current notation is what comes out from pandoc so I really don't think we want to change it without good cause.

N.B. In my Chrome and Firefox the L3 headings are a lot smaller than the L1. Also Safari it looks good.

https://drive.google.com/file/d/1ulJ6HvIgcqGuugsQOXIhXBSBF7r2eRro/view?usp=sharing